blob/blob.go

```diff
@@ -65,6 +70,10 @@ func (b *Blob) Validate() error {
 	return nil
 }
 
+func (b *Blob) Compare(other *Blob) int {
+	return bytes.Compare(b.RawNamespace(), other.RawNamespace())
+}
```

Comment:
Usually, such methods are called Equals with the bool return value, but as long as this is not PR targeting Blob we can omit that and this method altogether

Reply:
Ref: https://github.com/celestiaorg/go-square/pull/68#discussion_r1635996282

Reply:
Thats improvement over that comment

Reply:
I actually thinking comparing is more important because it's more common to want to order these things then see if the namespaces are equal

Reply:
Of course nothing wrong with having both